src/translate.rs

```diff
@@ -61,26 +61,22 @@ pub trait Ptr: Copy {
 }
 
 impl <T> Ptr for *const T {
-    fn is_null(&self) -> bool {
-        (*self).is_null()
-    }
+    #[inline]
+    fn is_null(&self) -> bool { (*self).is_null() }
```

Comment:
I guess, I've picked up this style from somewhere in the libstd or similar and it stuck.. :)